src/PokeList.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: academy
+ * Date: 14/11/2018
+ * Time: 11:48
+ */
+
+namespace Pokedex;
+
+use Pokedex\DbConnection;
+
+class PokeList {
+
+    private $list;
+
+    /**
+     * Creates a new pokeList using a DBConnection
+     * @param \PDO $connection The particular database object to use for retrieving the list of pokemon.
+     */
+    public function __construct(\PDO $connection)
+    {
+        $stmt = $connection->prepare('SELECT `id`, `name`, `type_1`, `type_2` FROM `pokemon`');
+        $stmt->execute();
+        $this->list = $stmt->fetchAll(\PDO::FETCH_CLASS, 'Pokedex\Pokemon');
+    }
+
+    /**
+     * @return array Returns an array containing the pokemon.
+     */
+    public function getPokemon() : array
```

Comment:
Consider renaming to be consistent with what you are 'getting' from the object i.e. getList rather than get getPokemon or rename the property to $pokemon.

Reply:
changed in 2511b60

I wrote it that way initially because pokemon doesn't have a plural and I wanted to avoid confusion between a singular pokemon and a group of pokemon, but was advised to make it consistent